pkg/util/ownership/backup_owner.go

```diff
@@ -0,0 +1,13 @@
+package ownership
+
+type Ownership struct {
```

Comment:
Add comment to describe the purpose of this struct?

Reply:
Done